### PR TITLE
Make example more illustrative

### PR DESCRIPTION
**Pull request by paulhenrich**
I'm at the Montevideo Elixir meetup, and we have some ideas for
improving the documentation. The idea here is to make the example
less abstract, and clarify the meaning of "3" by providing a
map with some context.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -52,9 +52,11 @@ import MyApp.Gettext
 gettext "Here is one string to translate"
 
 # Plural translation
-ngettext "Here is the string to translate",
-         "Here are the strings to translate",
-         3
+ngettext "The apple is ripe",
+         "The apples are ripe",
+         %{ thing: 'apple', size: 3 }.size
```
Comment (josevalim) [inline README.md]: Cosmetic detail: no spaces after `{` and before `}`. Thank you!

Comment (lexmag) [inline README.md]: Maybe `size` –> `count`?

**Comment by paulhenrich** [comment]
@lexmag @josevalim Thanks, I amended the commit with your suggested changes.

**Comment by josevalim** [comment]
:heart: :green_heart: :blue_heart: :yellow_heart: :purple_heart:

**Comment by whatyouhide** [comment]
My 2 cents: isn't using a map and then calling `.count` on it slightly more confusing than something like this:

```elixir
count = 3
ngettext "foo",
         "foos",
         count
```

**Comment by lexmag** [comment]
Agree with @whatyouhide.

**Comment by paulhenrich** [comment]
I'm fine with making that change. I agree it's a bit clearer.

**Pull request merged**